test/rust/cargo/cargo-test/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "cargo-test"
+version = "0.1.0"
+authors = ["smackers"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+num-traits = "0.2.12"
+smack = { path = "/usr/local/share/smack/lib/smack" }
```

Comment:
This is not ideal since it's a very platform-dependent path. Maybe we can do this: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories?

Reply:
Or at least we should put it somewhere that can be referred using an environment variable?

Reply:
I do like the idea of having it be a git repo. However, I think for simplicity, we should do it this way for now.